quarkus/service/src/main/java/org/apache/polaris/service/quarkus/auth/external/OidcTenantResolvingAugmentor.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.quarkus.auth.external;
+
+import io.quarkus.security.identity.AuthenticationRequestContext;
+import io.quarkus.security.identity.SecurityIdentity;
+import io.quarkus.security.identity.SecurityIdentityAugmentor;
+import io.quarkus.security.runtime.QuarkusSecurityIdentity;
+import io.smallrye.mutiny.Uni;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import org.apache.polaris.service.quarkus.auth.external.tenant.OidcTenantResolver;
+import org.eclipse.microprofile.jwt.JsonWebToken;
+
+/**
+ * A {@link SecurityIdentityAugmentor} that resolves the Polaris OIDC tenant configuration for the
+ * given identity and adds it as an attribute to the {@link SecurityIdentity}.
+ */
+@ApplicationScoped
+public class OidcTenantResolvingAugmentor implements SecurityIdentityAugmentor {
```

Comment:
I think a lot of these classes can be package private. Any reason to expose them outside of this package?

Reply:
This one specifically is referenced outside the package, but OK for the others 👍